net/pfSense-pkg-haproxy/files/usr/local/www/haproxy/haproxy_global.php

```diff
@@ -135,6 +135,7 @@
 					unset($haproxycfg['config'][0]['enable']);
 				}
 			} else {
+				$haproxycfg['config'][0] = array();
 				$haproxycfg['config'][0]['enable'] = 'off';
```

Comment:
Using our new access functions, this line would become:

```php
array_set_path($haproxycfg, 'config/0/enable', 'off');
```

That way is safe no matter what parts may be missing from the configuration data.